projects/packages/changelogger/tests/php/tests/src/UtilsTest.php

```diff
@@ -378,15 +377,15 @@ public function testGetRepoData() {
 		);
 		Utils::runCommand( array( 'git', 'init', '-b', 'main', '.' ), ...$args );
 		Utils::runCommand( array( 'git', 'add', 'in-git.txt' ), ...$args );
-		Utils::runCommand( array( 'git', 'commit', '-m', 'Commit (#123)' ), ...$args );
+		Utils::runCommand( array( 'git', 'commit', '--no-gpg-sign', '-m', 'Commit (#123)' ), ...$args );
```

Comment:
I'm guessing this is because your global git config is configured to sign. Probably we should instead figure out how to ignore the global config entirely.

Looks like setting `GIT_CONFIG_GLOBAL=/dev/null` and `GIT_CONFIG_NOSYSTEM=1` (above where we set other env vars) may do it.